Makefile

```diff
@@ -15,59 +17,62 @@ DOCTESTS = $(addsuffix /doctest,$(filter-out meta,${PACKAGES}))
 # All linting targets.
 LINT = $(addsuffix /lint,${PACKAGES})
 
-# Targets to build a distribution for each package.
-dist: ${DISTRIBUTIONS}
-${DISTRIBUTIONS} : %/dist : %/setup.py
-	cd $* \
-	&& python setup.py bdist_wheel \
-	&& twine check dist/*
 
-# Targets to run the test suite for each package.
-tests : ${TESTS}
-${TESTS} : %/tests :
+install:  ## Set up the project for development
+	poetry install --all-extras
+	poetry run pre-commit install
+
+dist:  ## Build the python package
```

Comment:
As someone not familiar with what "dist" means, I would prefer a more speaking name.